syntax = "proto3";

// message type for stm32 -> Unity communication
message pbPose {
    float x_position = 1;
    float y_position = 2;
    float yaw_angle = 3;
    float roll_angle = 4;
    float pitch_angle = 5;
    float steer_angle = 6;
    float rear_wheel_angle = 7;
}
